portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Google LLC
+//
```

Comment:
note that you don't really need to assign the copyright to Google, since you wrote this :)